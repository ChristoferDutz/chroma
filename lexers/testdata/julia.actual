% Implementing Feed Forward to calculate H

m = size(X, 1);
X = [ones(m, 1) X];
Z2 = Theta1 * X';
A2 = sigmoid(Z2);