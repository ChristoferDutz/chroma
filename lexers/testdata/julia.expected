[
  {"type":"Operator","value":"%"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Implementing"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Feed"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Forward"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"to"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"calculate"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"H"},
  {"type":"Text","value":"\n\n"},
  {"type":"Name","value":"m"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Name","value":"size"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"X"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"1"},
  {"type":"Punctuation","value":");"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"X"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"["},
  {"type":"Name","value":"ones"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"m"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"1"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"X"},
  {"type":"Punctuation","value":"];"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"Z2"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Name","value":"Theta1"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"*"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"X"},
  {"type":"Operator","value":"'"},
  {"type":"Punctuation","value":";"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"A2"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Name","value":"sigmoid"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"Z2"},
  {"type":"Punctuation","value":");"}
]
